plane/source/docs/apms-failsafe-function.rst

```diff
@@ -7,111 +7,85 @@ Plane Failsafe Function
+
+.. note:: having the receiver send a pre-set failsafe throttle value upon signal loss is NOT recommended and can lead to issues if a battery level failsafe is setup in QuadPlanes. Setting the receiver to send "no pulses" is much preferred. However, some very old receivers will only send a low throttle level in failsafe, see the section below, :ref:`Old Receivers <old_RX>`.
+
+The system must be setup such that the throttle channel's signal can go below :ref:`THR_FS_VALUE<THR_FS_VALUE>` (Default is 950), but still be above it for normal low throttle stick operation. Before doing the :ref:`RC Calibration<common-radio-control-calibration>` setup step which determines normal operating ranges for the throttle channel and others, make sure that the low throttle stick position on your transmitter is above :ref:`THR_FS_VALUE<THR_FS_VALUE>`. This can be done several ways:
+
+- When you do the :ref:`RC Calibration <common-radio-control-calibration>` setup step, change the trim tab for the throttle channel to adjust its signal 40-50us above :ref:`THR_FS_VALUE<THR_FS_VALUE>` at low throttle stick. This will be the normal operating position. Lowering the trim tab and setting the :ref:`THR_FS_VALUE<THR_FS_VALUE>` to that value allows initiating a failsafe at low trim.
+- Setup a transmitter switch that you will use to force failsafe such that it forces the throttle channel signal, using a mix, to below :ref:`THR_FS_VALUE<THR_FS_VALUE>` when activated, allowing normal operation otherwise.
+
+.. _fs_actions:
+
+RC Failsafe Operation
+~~~~~~~~~~~~~~~~~~~~~
+
+-  When RC Failsafe is entered, all RC inputs (except throttle in the case of Throttle Failsafe), are ignored as the autopilot takes its failsafe actions.
+-  First, the autopilot will go into Short Failsafe when it detects RC Failsafe for more than :ref:`FS_SHORT_TIMEOUT<FS_SHORT_TIMEOUT>` seconds.
+-  A message will be displayed on your Ground Control Station(GCS), or OSD, if its message panel is enabled, that a Short failsafe is active, and the flight controller will take the :ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>`, if enabled.  The default is CIRCLE mode.
+-  If the condition causing the Short Failsafe is removed, the vehicle will return to the previous mode, and a message will be displayed that Short Failsafe is OFF. If it was a Throttle Failsafe that caused the RC Failsafe, and throttle was increased to exit, then an additional message will be sent stating that the Throttle Failsafe is OFF.
+-  If the loss of signal is longer than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` seconds the autopilot will go into Long Failsafe, send a message to the GCS that it has been entered and execute the :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` action, if enabled. The default setting for long failsafe is RTL (Return to Launch).
+-  If the RC Failsafe condition is later exited, a message will be displayed that the Long Failsafe is OFF, but the mode will not revert.
```

Comment:
```suggestion
-  If the RC Long Failsafe condition is later exited, a message will be displayed that the Long Failsafe is OFF, but the mode will not revert.
```

Reply:
actually the original wording is correct RC Failsafe is a condition applicable to both short and long failsafe timeouts...this was talking about removing the RC Failsafe condition, while in a Long Failsafe status

Reply:
Ok, I thought line 65 was regarding Short Failsafe and line 67 was for Long Failsafe, as Short Failsafe does return to previous mode, but Long Failsafe does not, was just trying to make it more clear. Is possible I am misunderstanding the intent.